src/libImaging/ImagingPlatform.h

```diff
@@ -17,82 +21,37 @@
 #error Sorry, this library requires ANSI header files.
 #endif
 
-#if defined(PIL_NO_INLINE)
-#define inline
-#else
 #if defined(_MSC_VER) && !defined(__GNUC__)
 #define inline __inline
 #endif
-#endif
-
-#if defined(_WIN32) || defined(__CYGWIN__) /* WIN */
```

Comment:
I will examine this and return if needed:
https://github.com/python-pillow/Pillow/pull/5807